packages/data-transformer/src/transformers/data.transformer.spec.ts

```diff
@@ -0,0 +1,143 @@
+import "reflect-metadata"
+import {DataTransformer} from "./data.transformer";
+import {LowercaseNormalizer} from "../normalizers/lowercase.normalizer";
+import {DataTransformerBuilder} from "./data-transformer.builder";
+import {DataTransformerInterceptor} from "../interfaces/data-transformer-interceptor.interface";
+import {DataTransformerInterceptorUniqueKeyType} from "../types/data-transformer-interceptor-unique-key.type";
+import {DataTransformerRow} from "../types/data-transformer.row";
+import 'jest-extended';
+
+describe('Data Transformer', () => {
```

Comment:
If you have time I would also add a tests that validates that the data transformer throws as expected when errors occur.